db/migrate/20250922145129_add_green_deal_plans_scotland_table.rb

```diff
@@ -0,0 +1,22 @@
+class AddGreenDealPlansScotlandTable < ActiveRecord::Migration[8.0]
```

Comment:
I didn't think we need a separate Green Deal Plans table for Scotland?  As far as we know they should need the same data for disclosure, and the same updates to be made by the GDCC people, so probably easiest to reuse the existing `green_deal_plans` table but with a separate `scotland.green_deal_assessments` that links the plans to Scottish EPCs?